### ISSUE TITLE
New Entries (Sprout Forms) report with All in dropdown

### ISSUE THREAD

**Issue by espensgr** [barrelstrength/craft-sprout-reports#113]
i'm trying to add a report with all of the entries of a form, so there should be a choice in the dropdown for "All" that i without time constraints.

**Comment by BenParizek** [comment]
I agree @espensgr. I'll tag this as a FR and add it in a coming release.

In the meantime, as you've likely discovered, you can select the custom range and select a wider range than you need. I know that's not an ideal solution, but could save the trouble of updating it until we have an "All" option available.
